design-documents/graph-ql/coverage/CartAddressOperations.graphqls

```diff
@@ -0,0 +1,84 @@
+type Query {
+    addBillingAddressToCart(input: AddBillingAddressToCartInput): AddBillingAddressToCartOutput
+    addShippingAddressToCart(input: AddShippingAddressToCartInput): AddShippingAddressToCartOutput
+}
+
+input AddBillingAddressToCartInput {
+    address_id: Int
+    address: CartAddressInput!
+}
+
+input AddShippingAddressToCartInput {
+    address_id: Int # Can be provided in one-page checkout and is required for multi-shipping checkout
```

Comment:
Have we accounted for multishipping scenarios here? I don't yet see how we tie cart items with shipping addresses, in case there are multiple

Reply:
Done